support-workers/src/main/scala/com/gu/support/workers/lambdas/CreateZuoraSubscription.scala

```diff
@@ -316,12 +214,13 @@ object DigitalSubscriptionGiftRedemption {
     (for {
       ratePlan <- subscription.ratePlans.headOption
       productRatePlan <- catalogService.getProductRatePlanFromId(catalog.DigitalPack, ratePlan.productRatePlanId)
-      newEndDate = LocalDate.now()
+      startDate = LocalDate.now() // TODO needed for thank you email
```

Comment:
I have added TODOs where I have identified which values need to end up in the State in the next PR